clayui.com/plugins/gatsby-plugin-svg-compile-sprite/gatsby-node.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: © 2018 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// Compiles SVG icons in directory `clay-css/src/images/icons` into a spritemap
+
+const fs = require('fs');
+const path = require('path');
+
+const generateSvgSprite = (pluginOptions) => {
+	const REGEX_FILE_EXT_SVG = /(flags-|.svg$)/g;
+	const REGEX_HTML_COMMENTS = /<!--(.+?)(-->\n|-->)/gs;
+	const REGEX_SVG_VIEWBOX = /(\sv|v)iewBox="(.+?)"/g;
+	const REGEX_SVG_TAG = /(<\/svg|<svg(.+?))>(\n|)/g;
+
+	const filesArr = fs.readdirSync(pluginOptions.srcDir);
+
+	let strSprite = `<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">`;
+
+	filesArr.forEach((file) => {
+		let icon = fs
+			.readFileSync(path.join(pluginOptions.srcDir, file), 'utf8')
+			.toString();
```

Comment:
@pat270 we may have to ignore the `icons.svg` file, did you encounter any problems with it when you were running this?

Reply:
I didn't have any problems other than `icons.svg` showing up in git un-staged changes. I think it would be ok not to ignore it since we can diff the changes we made?

Reply:
Makes sense, maybe some people can add these files to the commit, by mistake.

Reply:
That's a good point, we might want to run the site on the latest release of Clay CSS instead of what is on the master branch. This would show un-staged changes in git. If you want to ignore it, we can do that. I think the case would be rare though.

Reply:
No problem, we can go without ignoring it and if there is a problem we can do it.